### ISSUE TITLE
Add test for components

### ISSUE THREAD

**Issue by xcv58** [xcv58/Tab-Manager-v2#65]

**Comment by xcv58** [comment]
Both jest and ava are not working out of box. Jest has this issue:
```
    [BABEL] /Users/xcv58/chrome-extensions/Tab-Manager-v2/test_setup.js: /Users/xcv58/chrome-extensions/Tab-Manager-v2/node_modules/babel-preset-react/lib/index.js.env is not supported in a preset
```

ava has webpack integration issue, can be solved by https://github.com/istarkov/babel-plugin-webpack-loaders

**Comment by xcv58** [comment]
We better to migrate babel env to latest https://github.com/babel/babel/tree/master/experimental/babel-preset-env 

or use create react app to replace existing webpack config.

**Comment by xcv58** [comment]
It may be solved by https://github.com/tleunen/babel-plugin-module-resolver

**Comment by xcv58** [comment]
Fixed by a series of commits around 85019c26c2b2d94e597b56e83f34006052b38896
Next step is improving test coverage.
